### PR TITLE
Add missing Python dependency for ceph-mgr

### PR DESCRIPTION
**Pull request by johscheuer**
We upgraded our Ceph cluster to `15.2.3` running on `18.04.5 LTS (Bionic Beaver)` which resulted in the following error/warning:

```bash
health: HEALTH_WARN
Module 'volumes' has failed dependency: No module named 'distutils.util'
```

In the previous version `14.2.11` there were no errors/warnings. We manually fixed this by installing `python3-distutils` and restarting the `ceph-mgr` but if the package is actual a requirement it makes sense to install it directly as dependency. The package is also available under Debian.

## Checklist
- [ ] References tracker ticket
- [x] Updates documentation if necessary
- [x] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: debian/control
Comment (tchaikov) [inline debian/control:223]: nit, could you move this up before `python3-jwt,` the they are ordered alphabetically?

Comment (johscheuer) [reply debian/control:223]: Thanks for the pointer, I somehow missed this completely. I adjusted the order.

**Review by tchaikov — CHANGES_REQUESTED** [review]
> mgr: Add missing Python dependency for ceph-mgr

also, could you s/mgr/debian/ in the title of commit message. and might want to be more specific regarding to the root cause:

> mgr/volumes/fs/operations/pin_util.py imports `distutils.util` for using `strtobool`

**Comment by johscheuer** [comment]
jenkins test api

**Review by tchaikov — APPROVED** [review]

**Comment by tchaikov** [comment]
this change should be backported.

**Comment by johscheuer** [comment]
Backported means in this context adding this commit to the `octopus` branch right and then open an additional PR?
I can open a PR for this later the day.

**Comment by tchaikov** [comment]
> Backported means in this context adding this commit to the `octopus` branch right and then open an additional PR?
> I can open a PR for this later the day.

@johscheuer hi Johannes, yeah. we need to get the change targeting master merged first for backporting the fix to a release branch. in general, a tracker ticket is created for tracking this progress. and our stable release team can take care of this once the fix lands on master branch. so instead of creating a PR, i'd suggest create a tracker ticket on https://tracker.ceph.com and add a 

```
Fixes: https://tracker.ceph.com/issues/<id>
```

right before your Signed-off-by line in the commit message. and leave the backport to us.

**Comment by tchaikov** [comment]
https://pulpito.ceph.com/kchai-2020-09-26_01:51:27-rados-wip-kefu-testing-2020-09-25-2324-distro-basic-smithi/

**Comment by tchaikov** [comment]
backport PR: #37422

**Pull request merged**
